src/core/issues-notes.js

```diff
@@ -118,7 +118,37 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          var labelListSpan = document.createElement('span');
```

Comment:
If it’s a list, then we might want a `ul`, but I don’t think we want a list at all

Reply:
If you want an arbitrary container, use a DocumentFragment.